common/djangoapps/edxmako/tests.py

```diff
@@ -41,6 +41,20 @@ def test_marketing_link(self):
             link = marketing_link('ABOUT')
             self.assertEquals(link, expected_link)
 
+        # test both variants of the 'courses' expected links
+        with patch.dict('django.conf.settings.FEATURES', {'ENABLE_MKTG_SITE': False}):
+            with patch.dict('django.conf.settings.MKTG_URL_LINK_MAP', {'COURSES': 'courses'}):
+                expected_link = reverse('courses')
+                link = marketing_link('COURSES')
+                self.assertEquals(link, expected_link)
+            # since /courses no longer exists for CME, we redirect to the medical school online classes
+            with patch.dict('django.conf.settings.MKTG_URL_LINK_MAP',
+                            {'COURSES': 'https://med.stanford.edu/cme/learning-opportunities/online.html'}):
```

Comment:
We should use a fake, arbitrary URL instead of a live one: `https://example.com`.